__tests__/tasks/MembershipRecompute.js

```diff
@@ -88,4 +88,35 @@ describe('task membership:recompute', () => {
     await run({ memberId: 1 })
     expect(spy).toHaveBeenCalledWith([1])
   })
+
+  test('should not send email to existing members', async () => {
+    const spy = jest.spyOn(ah.api.tasks.tasks['membership:recompute'], 'sendWelcomeMail')
+    await prepare({ memberId: 1, payments: [{ paymentDate: '2018-10-21' }] })
```

Comment:
you don't need to pass parameters for the test, actually the only parameter that makes sense is `cardId` and that would make even stronger message that `member.cardId` is not null.